python-backend/app/commands.py

```diff
@@ -75,3 +77,11 @@ def _create_data(num):
     def run_etl():
         from app.cli_jobs import ETL_jobs
         ETL_jobs.run_ETL()
+
+    @app.cli.command()
+    @click.option('-c', '--commit')
+    def run_append_tailings_reports_to_code_required_reports_then_destroy_tailings_data(commit):
```

Comment:
This is the most monstrous command line argument I've ever read. The extended description for this function should be provided in a pydoc/click description.

Also, lets split this into two commands just to be safe. Migrate *then* truncate.

Reply:
Given that we can nuke the CRR data if something goes wrong, that's a good idea, i didn't think there was a recoverable state inbetween the two steps.